addons/account_peppol/views/res_config_settings_views.xml

```diff
@@ -5,164 +5,163 @@
         <field name="model">res.config.settings</field>
         <field name="inherit_id" ref="account.res_config_settings_view_form"/>
         <field name="arch" type="xml">
-            <block id="analytic" position="after">
-                <block title="PEPPOL Electronic Document Invoicing">
-                <field name="is_account_peppol_eligible" invisible="1"/>
-                <div invisible="not is_account_peppol_eligible">
-                    <div id='account_peppol'>
-                        <div class="col-12 col-lg-12 o_setting_box">
-                            <field name="account_peppol_proxy_state" invisible="1"/>
-                            <div class="o_setting_left_pane">
-                                <field name="is_account_peppol_participant"
-                                       invisible="account_peppol_proxy_state != 'not_registered'"/>
-                            </div>
-                            <div class="o_setting_right_pane">
-                                <div invisible="account_peppol_proxy_state != 'not_registered'">
-                                    <label for="is_account_peppol_participant" string='Use PEPPOL Invoicing'/>
-                                    <span class="fa fa-lg fa-building-o" title="Values set here are company-specific."/>
-                                    <div class="text-muted">
-                                        Start sending and receiving documents via Peppol as soon as your registration is complete.
-                                    </div>
-                                    <div class="alert alert-warning mt-3"
-                                         role="alert"
-                                         invisible="not is_account_peppol_participant or not account_peppol_endpoint_warning">
-                                        <field name="account_peppol_endpoint_warning"/>
+            <div id="account_peppol" position="replace">
+                <field name="is_any_account_peppol_participant" invisible="True"/>
```

Comment:
```suggestion
                <field name="is_any_account_peppol_participant" invisible="1"/>
```
(to stay consistent with the rest of the code)

Reply:
~True~ 1 😉